### PR TITLE
Page to sign account file

### PR DESCRIPTION
**Pull request by vitormattos**

### CODE REVIEW

File: appinfo/routes.php [hunk 1]
```diff
@@ -68,6 +68,8 @@
 		// Pages - public
 		['name' => 'page#sign',                       'url' => '/p/sign/{uuid}', 'verb' => 'GET'],
 		['name' => 'page#sign',                       'url' => '/p/sign/{uuid}/{path}', 'verb' => 'GET', 'requirements' => ['path' => '.+'], 'postfix' => 'extra'],
+		['name' => 'page#signAccountFile',            'url' => '/p/accout/files/approve/{uuid}', 'verb' => 'GET'],
+		['name' => 'page#signAccountFile',            'url' => '/p/accout/files/approve/{uuid}/{path}', 'verb' => 'GET', 'requirements' => ['path' => '.+'], 'postfix' => 'extra'],
```
Comment (vinicius73) [inline appinfo/routes.php:72]: **typo:** account

**Review by vinicius73 — CHANGES_REQUESTED** [review]

File: appinfo/routes.php (hunk 1 above)
Comment (vitormattos) [inline appinfo/routes.php:72]:
```suggestion
		['name' => 'page#signAccountFile',            'url' => '/p/account/files/approve/{uuid}', 'verb' => 'GET'],
		['name' => 'page#signAccountFile',            'url' => '/p/account/files/approve/{uuid}/{path}', 'verb' => 'GET', 'requirements' => ['path' => '.+'], 'postfix' => 'extra'],
```

**Pull request merged**